src/taco-utils/tacoPackageLoader.ts

```diff
@@ -93,13 +93,13 @@ module TacoUtility {
     }
 
     export class TacoPackageLoader {
+        public static FILE_REGEX_PREFIX: string = "file://";
```

Comment:
The reason I made it public is it needs to be defined before FILE_URI_REGEX (which is public so other classes can use it), and if it is private then tslint complains that I have a private variable declared before a public one.